src/templates/example.html

```diff
@@ -34,6 +34,14 @@
         </div>
       </div>
       {{/sections}}
+      <div style="text-align: center">
+        <div style="display: inline-block; margin: 0 auto;">
+          <a href="{{prev}}">Prev</a>
```

Comment:
I'd prefer to only have the next element (the previous is not relevant when going through the samples.